DigitalLearningSolutions.Web/ViewModels/LearningPortal/RecommendedLearning/SearchableRecommendedResourceViewModel.cs

```diff
@@ -40,5 +42,12 @@ public SearchableRecommendedResourceViewModel(RecommendedResource recommendedRes
         public bool IsCompleted { get; set; }
 
         public int? LearningLogItemId { get; set; }
+
+        public string Rating { get; set; }
+
+        private string GetRecommendationRatingString(decimal recommendationScore)
+        {
+            return recommendationScore >= 100 ? "Essential" : recommendationScore >= 40 ? "Recommended" : "Optional";

```

Comment:
Is there any reason why this can't be the definition of Rating instead of having this extra function and needing to call it?